airflow/www/views.py

```diff
@@ -933,10 +938,13 @@ def _iter_parsed_moved_data_table_names():
             permissions.RESOURCE_ADMIN_MENU,
         ) in user_permissions and conf.getboolean("webserver", "warn_deployment_exposure"):
             robots_file_access_count = (
-                session.query(Log)
-                .filter(Log.event == "robots")
-                .filter(Log.dttm > (utcnow() - datetime.timedelta(days=7)))
-                .count()
+                select(Log)
+                .where(Log.event == "robots")
+                .where(Log.dttm > (utcnow() - datetime.timedelta(days=7)))
+                # .count()
```

Comment:
```suggestion
```

Reply:
done

Reply:
Not sure what happened but it's still there

Reply:
yet to commit the change :) still running tests on local

Reply:
Please check [c56bafc](https://github.com/apache/airflow/pull/32177/commits/c56bafc4f4ef7f68819b4901081cb2ad1112e461)